src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -35,10 +35,10 @@ angular
  * There is an example below of how this should look.
  *
  * ### Notes
- * The `md-autocomplete` uses the the [VirtualRepeat](/api/directive/mdVirtualRepeatContainer)
+ * The `md-autocomplete` uses the the [VirtualRepeat](api/directive/mdVirtualRepeat)
```

Comment:
Ok, well I added links on both the VirtualRepeat and the VirtualRepeatContainer to easily jump between them.

I think seeing the docs for the VirtualRepeat is more useful than the container?